web/taskmaster/eslint.config.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+const eslint = require("@eslint/js");
+const tseslint = require("typescript-eslint");
+const angular = require("angular-eslint");
+  
+module.exports = tseslint.config(
```

Comment:
Did you try using the schematics for installation? Feels like you did it a little bit more manual 😁 
https://github.com/angular-eslint/angular-eslint?tab=readme-ov-file#quick-start

Reply:
I ran `ng lint` and then it asked to install schematics 👀

Reply:
ohh I see, I ususally install eslint before trying to run the script. So I guess it's a new way recommended by the framework.